test/ServerComparison.TestSites/Program.cs

```diff
@@ -18,6 +19,11 @@ public static void Main(string[] args)
 
             var builder = new WebHostBuilder()
                 .UseConfiguration(config)
+                .ConfigureLogging((_, factory) =>
+                {
+                    factory.AddConsole();
+                    factory.AddFilter("Console", level => level >= LogLevel.Warning);
```

Comment:
This is going to need a better API. `factory.AddLevelFilter("Console", LogLevel.Warning);`